src/components/EmptyContent/EmptyContent.vue

```diff
@@ -57,8 +62,8 @@ export default {
 
 <template>
 	<div class="empty-content" role="note">
-		<div class="empty-content__icon" :class="icon" role="img">
-			<!-- @slot Optional icon slot -->
+		<div v-if="hasIcon" class="empty-content__icon">
+			<!-- @slot Mandatory material design icon slot -->
 			<slot name="icon" />
 		</div>
 		<h2 class="empty-content__title">
```

Comment:
```suggestion
		<h2 v-if="hasTitle" class="empty-content__title">
```

Reply:
For https://github.com/nextcloud/nextcloud-vue/issues/2500

I'd probably not make it mandatory then for cases where the empty content is used below any other heading than h1